CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+### Added
+
+- Add optional parameter `enableWebAudio: boolean` in the config of meeting manager to decide whether to enable web audio in the device controller.
+- Extend the ability to accept device and audioTransformDevice as audio input in meeting manager.
```

Comment:
nit: devce -> Device and audioTransformDevice -> AudioTransFormDevice.
Should we also call out the change to `useSelectAudioInputDevice()` hook here?

Reply:
Updated the type, and added the change of `useSelectAudioInputDevice()` hook in the `Changed` part